lib/resources/service.rb

```diff
@@ -134,9 +134,9 @@ def select_service_mgmt # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticCom
         else
           SysV.new(inspec, service_ctl || '/usr/sbin/service')
         end
-      elsif %w{redhat fedora centos}.include?(platform)
+      elsif %w{redhat fedora centos oracle}.include?(platform)
```

Comment:
maybe we spit this and support more redhat systems?

```
elsif os.redhat?
  if os[:release].to_i >= 7
    Systemd.new(inspec, service_ctl)
  else
    SysV.new(inspec, service_ctl || '/sbin/service')
  end
elsif %w{fedora}.include?(platform)
  if os[:release].to_i >= 15
    Systemd.new(inspec, service_ctl)
  else
    SysV.new(inspec, service_ctl || '/sbin/service')
  end
```

to support https://github.com/chef/train/blob/master/lib/train/extras/os_detect_linux.rb#L17